lib/ama_validators/name_format_validator.rb

```diff
@@ -0,0 +1,7 @@
+class NameFormatValidator < ActiveModel::EachValidator
+  def validate_each(object, attribute, value)
+    unless value =~ /\A[a-zA-ZÀàÂâÄäÈèÉéÊêËëÎîÏïÔôŒœÙùÛûÜüŸÿÇç,.'- ]+\z/
```

Comment:
I think space has to be ```\s```. You might have to put it at the start of the group too. ```/\A[\sa-z...```